### PR TITLE
Support 'GetDeviceInfo' action

### PR DESCRIPTION
**Pull request by overcyn**
Required for https://github.com/CareEvolution/RKStudio-Participant/pull/889

### CODE REVIEW

**Comment by mmertsock** [comment]
This approach has some problems now that we're adding support for multiple message types in our apps. It tightly couples ResearchKit to the features of the app using ResearchKit.

An alternative approach: add an NSArray property to ORK[1]WebViewStepViewController.h that is the list of message names that should be passed to the scriptMessageHandler. Then in the .m files, just loop over that array of names and for each name do `[_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:name];`

So our apps, when creating ORK[1]WebViewStepViewControllers (in the viewControllerForStep delegate method), it would set the array of supported names at the same time it sets the scriptMessageHandler property.

**Comment by overcyn** [comment]
Thats a good point. Added a property called messageNames to ORK[1]WebViewStepViewController.

File: ResearchKit/Common/ORKWebViewStepViewController.m [hunk 1]
```diff
@@ -134,6 +134,7 @@ - (instancetype)initWithStep:(ORKStep *)step {
         [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"ResearchKit"];
         [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"GetAccessToken"];
         [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"GetDelegatedAccessToken"];
+        [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"GetDeviceInfo"];
```
Comment (mmertsock) [inline ResearchKit/Common/ORKWebViewStepViewController.m]: GetAccessToken/GetDelegatedAccessToken/GetDeviceInfo should be removed here, to match the updated ORK1WebViewStepViewController.m, right?

Comment (overcyn) [reply ResearchKit/Common/ORKWebViewStepViewController.m]: Fixed

**Review by mmertsock — APPROVED** [review]
LGTM 🚢

**Pull request merged**